server/router/vehicle.ts

```diff
@@ -2,37 +2,40 @@ import express from 'express';
 import { v4 as uuid } from 'uuid';
 import * as db from './common';
 import { Vehicle } from '../models/vehicle';
+import { validateUser } from '../util';
 
 const router = express.Router();
 const tableName = 'Vehicles';
 
 // Get a vehicle by id
-router.get('/:id', (req, res) => {
+router.get('/:id', validateUser('User'), (req, res) => {
   const { params: { id } } = req;
   db.getById(res, Vehicle, id, tableName);
 });
 
 // Get all vehicles
-router.get('/', (req, res) => db.getAll(res, Vehicle, tableName));
+router.get('/', validateUser('User'), (req, res) => {
```

Comment:
Don't know if this should be Dispatcher? Don't think there is any reason we should let everyone see the info of every vehicle since they already will have access to info on the specific vehicles they need at any given time.

Reply:
yeah that's true